extensions/ulid/package.json

```diff
@@ -0,0 +1,75 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "ulid",
+  "title": "ULID",
+  "description": "some basic ulid tools",
+  "categories": [
+    "Developer Tools"
+  ],
+  "icon": "logo_512.png",
+  "author": "entwine7250",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "generator",
+      "title": "Generate ULID by now",
+      "description": "generate a ULID id by current time",
+      "mode": "no-view"
+    },
+    {
+      "name": "decode",
+      "title": "Translate ULID to Time",
```

Comment:
It would be good to indicate that you expect the ULID to be on the user's clipboard. Same applies to the error message for when the content of the clipboard is not an ULID.